src/ResourceManagement/CosmosDB/Domain/CosmosDBAccount/Definition/IDefinition.cs

```diff
@@ -214,8 +224,9 @@ public interface IWithWriteReplication
         /// A georeplication location for the CosmosDB account.
```

Comment:
> Sets a write replication location ...

And above `WithReadReplication` should have correct summary as well.

Similar for IUpdate.cs